content/en/getting-started/building-and-upgrading.md

```diff
@@ -73,7 +88,7 @@ The answer to this will depend on your setup but here are two things to keep in
 
 **What if I don't want to base my image on the Kairos released artifacts?**
 
-No problem, just [build your image from scratch](/reference/build-from-scratch/)
+No problem, just [build your image from scratch](/reference/kairos-factory/)
```

Comment:
This doesn't sync with what's above. A ove we used to say, get a published artifact then install ruby on it but now it makes it use kairos-init, thus pointing here again to Kairos init makes it weird